packages/zarathustra/skills/asylum_abci_app/behaviours.py

```diff
@@ -94,18 +120,11 @@ def __init__(self, **kwargs: Any) -> None:
         self._event = None
         self._is_done = False  # Initially, the state is not done
 
-        self.data_dir = Path(__file__).parent / "data"
+        self.data_dir = get_repo_root() / "data"
```

Comment:
we should make this a default action, where we do this if we do not pass the directory for the data. 
When we are in docker